tests/components/kaleidescape/conftest.py

```diff
@@ -17,7 +17,7 @@
 
 
 @pytest.fixture(name="mock_device")
-def fixture_mock_device() -> Generator[AsyncMock]:
+def fixture_mock_device() -> Generator[MagicMock]:
```

Comment:
Any particular reason to change it to `MagicMock`? AFAICT most methods are async, so `AsyncMock` would make sense IMO.

Reply:
My understanding is that `AsyncMock` is for when you are patching an individual async function/method.
In this case, it is not a method or a function that is being patched, but the class creation.

Reply:
Makes sense. Thanks for the explanation!